packages/jaeger-ui/src/model/ddg/PathElem.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
```

Comment:
No a fan of disabling - can we remove the aliasing instead? And if we must disable, it's should be localized only to where needed, not the whole file.

Reply:
Ok Understood